shared/src/iosMain/kotlin/com/br/kmmdemo/Platform.ios.kt

```diff
@@ -1,8 +1,15 @@
 package com.br.kmmdemo
 
+import androidx.compose.runtime.Composable
+import kotlinx.cinterop.ExperimentalForeignApi
 import platform.UIKit.UIDevice
+import platform.UIKit.UIScreen
 
 actual class Platform actual constructor() {
     actual val name: String =
         UIDevice.currentDevice.systemName() + " " + UIDevice.currentDevice.systemVersion
+
```

Comment:
Same comment here as above.  We can probably just call this platform logic directly where it's used as that would already be platform specific code.